code/modules/mob/living/damage_procs.dm

```diff
@@ -264,7 +264,7 @@
 		update_stamina()
 
 // damage MANY bodyparts, in random order
-/mob/living/proc/take_overall_damage(brute = 0, burn = 0, stamina = 0, updating_health = TRUE)
+/mob/living/proc/take_overall_damage(brute = 0, burn = 0, stamina = 0, updating_health = TRUE, only_robotic = FALSE, only_organic = FALSE)

```

Comment:
should probably just make this one var that passes a status type, that way if we ever add more we won't have to add more arguments.